src/triggers/createPullRequestForJiraIssue.ts

```diff
@@ -38,13 +38,15 @@ import { PullRequestForIssueTemplate, render } from '@sr-services/Template'
  * --data    '{"ref": "develop", "inputs": { "email": "dave@shuttlerock.com", "event": "createPullRequestForJiraIssue", "param": "STUDIO-232" }}' \
  * https://api.github.com/repos/Shuttlerock/actions/actions/workflows/trigger-action.yml/dispatches
  *
- * @param {string} email    The email address of the user who will own the pull request.
- * @param {string} issueKey The key of the Jira issue we will base the pull request on.
+ * @param {string} email       The email address of the user who will own the pull request.
+ * @param {string} userCommand The command given by the user of the Jira issue we will base the pull request on.
  */
 export const createPullRequestForJiraIssue = async (
   email: string,
-  issueKey: string
+  userCommand: string
 ): Promise<void> => {
+  const [issueKey, repositoryName] = userCommand.trim().split(/\s+/)
```

Comment:
```suggestion
  issueKey: string
): Promise<void> => {
  const [issueKey, repositoryName] = userCommand.trim().split(/\s+/)
  const repositoryName = process.env['GITHUB_REPOSITORY']
```

@DanielM-SR I think to use the env-var it would be as simple as this ☝️ What you have here will work too though 👍

Reply:
Oh, I was definitely overthinking it then. 😆